src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/TestRunnerTests.cs

```diff
@@ -257,15 +266,22 @@ public async Task TestErrorOverride()
             // Test override BaseRunner.IsError
             var runner = new MockErrorRunner
             {
-                _hook = (expr, setup) => FormulaValue.New(1),
-                _isError = (value) => true
+                _hook = (expr, setup) => 
+                    expr switch {
+                        "1" => FormulaValue.New(1),
+                        "IsError(1)" => FormulaValue.New(true),
```

Comment:
It would be good to also assert that `IsError(1)` was called in this test (i.e., initialize a variable as false, set it to true in this case, then assert that it's true at the end)

Reply:
I'll add a 2nd test that returns IsError() false, and verifies a failure. That should cover this.